requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
```

Comment:
think there's a lot more than this:
wikipedia
flask

there's other ones but I'm forgetting them